lumicks/pylake/fitting/models.py

```diff
@@ -259,6 +259,43 @@ def odijk(name):
     )
 
 
+def dsdna(name, kilobase_pairs, kilobase_pairs_per_micron=0.34, temperature=24.53608821):
+    """Odijk's Extensible Worm-Like Chain model with distance as the dependent variable, using
+    user-specified kilobase-pairs (useful for 10 pN < F < 30 pN).
+
+    References:
+      1. T. Odijk, Stiff Chains and Filaments under Tension, Macromolecules
+         28, 7016-7018 (1995).
+      2. M. D. Wang, H. Yin, R. Landick, J. Gelles, S. M. Block, Stretching
+         DNA with optical tweezers., Biophysical journal 72, 1335-46 (1997).
+
+    Parameters
+    ----------
+    name : str
+        Name for the model. This name will be prefixed to the model parameter names.
+    kilobase_pairs : integer
+        The number of kilobase-pairs in the sample
+    kilobase_pairs_per_micron: float
+        The number of kilobase-pairs evaluating to 1 micron. This is used to convert the kilobase-pairs to microns for the fit function.
+    temperature: float
+        The temperature in celsius. This is used to calculate the boltzmann * temperature value
+    """
+    import scipy.constants as spy
+
+    model = odijk(name)
+    model.defaults[f"{name}/Lc"].value = kilobase_pairs * kilobase_pairs_per_micron
+    # Based on lambda dna C-trap experiments
+    model.defaults[f"{name}/Lp"].value = 56.0
+
+    # Convert temperature to kelvin in order to evaluate kT
+    temp = spy.convert_temperature(temperature, "C", "K")
+    # Conversion factor required to convert kT value from J/K to pN*nm/K
+    conversion_factor = 1e21
+    boltzmann = spy.k
+    model.defaults["kT"].value = conversion_factor * boltzmann * temp
```

Comment:
Personally, I don't really see much added value in assigning these to variables first.
```python
# Conversion of 1e21 required to convert kT value from J/K to pN*nm/K
model.defaults["kT"].value = 1e21 * constants.k * temp
```